hamster_gtk/hamster_gtk.py

```diff
@@ -133,7 +137,7 @@ class HamsterGTK(Gtk.Application):
 
     def __init__(self):
         """Setup instance and make sure default signals are connected to methods."""
-        super(HamsterGTK, self).__init__()
+        super(HamsterGTK, self).__init__('org.projecthamster.hamster-gtk')
```

Comment:
This won’t work since Python constructors are not the GTK constructors but the GObject ones. Here it even errors with GObject.__init__() takes arguments. GTK constructor is the [`new`](https://lazka.github.io/pgi-docs/index.html#Gtk-3.0/classes/Application.html#Gtk.Application.new) method, which calls the GObject constructor, **[maybe](https://git.gnome.org/browse/gtk+/tree/gtk/gtkapplication.c?h=3.22.16#n715)** kwarg would work?